identity-iota/src/did/doc/iota_document.rs

```diff
@@ -388,6 +393,18 @@ impl IotaDocument {
     Ok(())
   }
 
+  pub fn attach_method_relationship(&mut self, did_url: IotaDIDUrl, relationship: MethodRelationship) -> Result<bool> {
```

Comment:
Nit: doc comment public functions, even if they just match `CoreDocument`.

```suggestion
  /// Attaches the relationship to the given method, if the method exists.
  ///
  /// Note: The method needs to be in the set of verification methods,
  /// so it cannot be an embedded one.
  pub fn attach_method_relationship(&mut self, did_url: IotaDIDUrl, relationship: MethodRelationship) -> Result<bool> {
```